docs/fundamentals/systems/core/SlotMixin.md

```diff
@@ -4,7 +4,7 @@ The SlotMixin is made for solving accessibility challenges that inherently come
 Until [AOM](https://wicg.github.io/aom/explainer.html) is in place, it is not possible to create relations between different shadow doms.
 The need for this can occur in the following situations:
 
-1. A user defined slot. For instance:
+- A user defined slot. For instance:
```

Comment:
Jut a detail:
"user-defined" could be a better way to write it, eg https://en.wikipedia.org/wiki/User-defined_function

Reply:
True, will update it 👍